setup.py

```diff
@@ -0,0 +1,65 @@
+from setuptools import setup, find_packages
+from io import open
+from os import path
+import pathlib
+
+# The directory containing this file
+HERE = pathlib.Path(__file__).parent
+
+# The text of the README file
+README = (HERE / "README.md").read_text()
+
+# automatically captured required modules for install_requires in requirements.txt
+with open(path.join(HERE, 'requirements.txt'), encoding='utf-8') as f:
+    all_reqs = f.read().split('\n')
+
+install_requires = [x.strip() for x in all_reqs if ('git+' not in x) and (
+    not x.startswith('#')) and (not x.startswith('-'))]
+dependency_links = [x.strip().replace('git+', '') for x in all_reqs \
+                    if 'git+' not in x]
+setup (
+ name = 'tasky',
+ description = 'cli for google tasks',
+ version = '1.0.0',
+ packages = find_packages(where='src'),
+ package_dir={'': 'src'}, 
+ install_requires = install_requires,
+ python_requires='>=3.6, <4', # any python greater than 2.7
+ entry_points='''
+        [console_scripts]
+        tasky=tasky.__main__:main
+    ''',
+ author="someone",
```

Comment:
It's probably fine to leave out `author` and `author_email` (lower down) since many have contributed to this over time.